docs/api/app.md

```diff
@@ -396,6 +396,15 @@ You can request the following paths by the name:
 * `videos` Directory for a user's videos.
 * `pepperFlashSystemPlugin`  Full path to the system version of the Pepper Flash plugin.
 
+### `app.getFileIcon(path, [options], callback)`
```

Comment:
When did this move from `NativeImage.from` to an `app` API.  Personally I liked it better the other way round.  The `app` module is already polluted as it is and IMO it shouldn't just become a dumping ground for new methods 👍